frontend/web/src/features/bookingengine/listingdetails/views/RangeCalendar.jsx

```diff
@@ -18,13 +18,16 @@ const weekdayLabels = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"];
 const monthLabel = (d) =>
   d.toLocaleString(undefined, { month: "long", year: "numeric" });
 
-function MonthGrid({ viewMonth, rangeStart, rangeEnd, onPick }) {
+function MonthGrid({ viewMonth, rangeStart, rangeEnd, onPick, pricing }) {
   const y = viewMonth.getFullYear();
   const m = viewMonth.getMonth();
```

Comment:
```suggestion
  const year = viewMonth.getFullYear();
  const month = viewMonth.getMonth();
```
Using year and month makes the code clearer, since y and m are not very clear variable names.

Reply:
nice observation, +1 this